theopensystemslab/planx-new
# feat: create send events when a payment request is marked as `paid_at`

Submits Invite to Pay applications when the `payment_request` is marked as `paid_at`. Relies on each Send component destination being able to send based on solely a `sessionId`. 

To test:
- go through a published flow with invite to pay feature flag enabled & toggled on in the Pay component
  - when you reach pay, click "invite someone else" to generate a payment request
  - pay for your application, confirm your `payment_request` now shows `paid_at`
- check that Hasura successfully "setup-payment-send-events" (1 event record independent of number of Send destinations): https://hasura.1709.planx.pizza/console/events/data/setup_payment_send_events/processed
- check that "One-off scheduled events" succeeded (1 event record _per_ Send destination): https://hasura.1709.planx.pizza/console/events/one-off-scheduled-events/processed
